### ISSUE TITLE
Expression parsing in classinfo placeholders

### ISSUE THREAD

**Issue by UltraFaceguy** [SkriptLang/Skript#59]
The following works:
drop 10 xp at location of player

The following does not:
set {_dank} to 420
drop {_dank} xp at location of player

Just kinda annoying and impeding progress on something I'm trying to do.

**Comment by bensku** [comment]
Skript parser is having serious problems with multiple variable types accepted. For example, in particle system it accepts either entity or location (for location or entity particles). As long as you don't use variables (or use <code>{var} as type</code>; should work too), it works just fine. However, if you **do** use variables, Skript somehow forgets type of the variable and tries to convert locations to entities; obviously that doesn't work at all.

Tl;dr: I need to fix Skript parser before this.

**Comment by Tuke-Nuke** [comment]
`Experience` is a type, so you need to set the variable to its type
```
set {_dank} to 420 xp
#or
set {_dank} to "420 xp" parsed as experience
drop {_dank}
```

**Comment by FranKusmiruk** [comment]
Closing this as it isn't realistically possible without a rework of how classinfos are parsed and to be honest, it kinda defeats the purpose of a classinfo in a way.
